domain/historical_stats.go

```diff
@@ -77,5 +77,10 @@ func (w *HistoricalStatsWorker) DumpHistoricalStats(tableID int64, statsHandle *
 
 // GetOneHistoricalStatsTable gets one tableID from channel, only used for test
 func (w *HistoricalStatsWorker) GetOneHistoricalStatsTable() int64 {
```

Comment:
Is this function only used in testing? If not, I'm afraid the `-1` can cause some panic.